roles/install_agent/tasks/Windows.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: "Windows: Include Windows.yml vars"
+  ansible.builtin.include_vars:
+    file: Windows.yml
+
+- name: "Windows: Remove windows defender feature on Windows Server OS"
+  ansible.windows.win_feature:
+    name: Windows-Defender
+    state: absent
+  register: win_feature_remove
+  when: "'Windows Server' in ansible_os_name"
+
+- name: "Block: Windows: Install SentinelOne agent"
+  when: not agent_installed
+  block:
+    - name: "Windows: Copy agent package to remote server"
+      ansible.windows.win_copy:
+        src: "{{ return_download_agent.original_message.full_path }}"
+        dest: "{{ remote_pkg_path }}"
+        mode: "0644"
+
+    - name: "Windows: Install agent package {{ return_download_agent.original_message.filename }}"
+      ansible.windows.win_package:
+        path: "{{ remote_pkg_path }}"
+        creates_service: "SentinelAgent"
+        arguments: "{{ exe_parameters if pkg_format == 'exe' else msi_parameters }}"
+      register: installation_result
+      no_log: "{{ hide_sensitive }}"
+      when: pkg_format == "msi" or pkg_format == "exe"
+
+    - name: "Windows: Wait for 15 seconds"
+      ansible.builtin.pause:
```

Comment:
This could lead to problems depending on the Windows System to be configured.

2 ideas to improve this: 
a. set time duration to wait variable and 15 sec as default
b. Would be ideal, but I'm not sure regarding implementability: Use win_wait or win_service_info module to check if desired sentinel one service is running after installation

Reply:
I've seen that you are checking with "win_service_info" for the availability of the Service shortly after. 

Maybe instead of having a fixed wait it would be better to just work with "retries / until" the service is available and maybe timeout after a given period of time.

Reply:
Compromise with @mwester117: Fix this in a follow up release

![image](https://github.com/svalabs/sva.sentinelone/assets/45071484/851b41e9-271e-4913-a984-7fe749f990ee)